src/sqlite.rs

```diff
@@ -442,4 +450,77 @@ impl Sqlite {
 
         Ok(())
     }
+
+    async fn get_indexes(
```

Comment:
Note: We collect all names without information on whether an index is unique. This should be sufficient for most cases without adding additional complexity.